### PR TITLE
Dockerfile for infrastructure base image

### PR DESCRIPTION
**Pull request by MebinAbraham**
Added a Dockerfile to build the base infrastructure image.

The initial approach is to have a repository per base image.

### How to review
- You can either run the container locally and exec into it and try deploy an environment or alternatively you can fly the new pipeline found in the related PR on the concourse repo.

### CODE REVIEW

**Review by ajmaddaford — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# eq-concourse-base-images
\ No newline at end of file
+# eQ Infrastructure Build Image
+
+
```
Comment (JamesGardiner) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1 +1,11 @@
...
+This repo holds the Dockerfile used to build the infrastructure base image.
```
Comment (JamesGardiner) [inline README.md]:
```suggestion
This repo holds the Dockerfile used to build the eq infrastructure base image.
```

**Review by JamesGardiner — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,11 @@
...
+
+
+## Docker
+The Dockerfile can be built using:
+```bash
+docker build -t eq-infrastructure-build-image:latest .
```
Comment (JamesGardiner) [inline README.md:9]: We should be using `make build` as the build command in order to maintain consistency with other eq projects.

**Review by JamesGardiner — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (MebinAbraham) [reply README.md:9]: I don't think it's worth adding a Makefile just for this one command, especially when it is primarily not used for local dev.

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**